src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.cpp

```diff
@@ -331,11 +331,18 @@ HOSTFXR_UTILITY::ParseHostfxrArguments(
 
 std::optional<fs::path>
 HOSTFXR_UTILITY::GetAbsolutePathToDotnet(
+     const fs::path & applicationPath,
```

Comment:
It may be better to pass these by value than by ref as we set processPath=requestedPath immediately.

Reply:
prevent's extra copies